src/unit/test_util.c

```diff
@@ -11,6 +11,150 @@
 #include <linux/magic.h>
 #endif
 
+int test_stringmatch(int argc, char **argv, int flags) {
+    UNUSED(argc);
+    UNUSED(argv);
+    UNUSED(flags);
+
+    TEST_BEGIN();
+
+#define TEST_STRINGMATCH(p, s, expectCase, expectNocase)   \
```

Comment:
This macro is defined inside this test case. I think we should `undef` it at the end of the function to leave no trace.